js/SplitButton/Easy-customization.md

```diff
@@ -667,7 +667,7 @@ Configure the styles.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img4.png) 
+![Image Possition in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img4.png) 
```

Comment:
Check position spelling
Image position in JavaScript Split Button

Reply:
Corrected